### PR TITLE
OH2-251 | Fix visit filter not working correctly

### PR DESCRIPTION
**Pull request by SteveGT96**
See OH2-251.

### CODE REVIEW

File: src/components/accessories/opds/filter/OpdFilterForm.tsx [hunk 1]
```diff
@@ -311,11 +311,11 @@ export const OpdFilterForm: FC<IOpdFilterProps> = ({
   const newPatientOptions = [
     {
       label: t("opd.all"),
-      value: "",
+      value: "A",
     },
     {
       label: t("opd.newadmittance"),
```
Comment (mwithi) [inline src/components/accessories/opds/filter/OpdFilterForm.tsx:317]: can you please also change “New Admittance” into “New Attendance”?

Comment (SteveGT96) [reply src/components/accessories/opds/filter/OpdFilterForm.tsx]: Sure...

File: src/resources/i18n/de.json [hunk 2]
```diff
@@ -178,7 +178,7 @@
     "referralfrom": "Empfehlung von",
     "referralto": "Empfehlung an",
     "newpatient": "Anwesenheitstyp",
-    "newadmittance": "Neue Aufnahme",
+    "newattendance": "Neue Aufnahme",
```
Comment (mwithi) [inline src/resources/i18n/de.json:181]: When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```

File: src/resources/i18n/fr.json [hunk 3]
```diff
@@ -142,7 +142,7 @@
     "referralfrom": "Visite prescrite par une autre unité hospitalière",
     "referralto": "Visite prescrite de l'extérieur",
     "newpatient": "Type de visite",
-    "newadmittance": "Nouvelle hospitalisation",
+    "newattendance": "Nouvelle hospitalisation",
```
Comment (mwithi) [inline src/resources/i18n/fr.json:145]: When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```

File: src/resources/i18n/it.json [hunk 4]
```diff
@@ -178,7 +178,7 @@
     "referralfrom": "Riferito",
     "referralto": "Rinvio ad altra struttura",
     "newpatient": "TIpo visita",
-    "newadmittance": "Nuova visita",
+    "newattendance": "Nuova visita",
```
Comment (mwithi) [inline src/resources/i18n/it.json:181]: this is ok

File: src/resources/i18n/sq.json [hunk 5]
```diff
@@ -178,7 +178,7 @@
     "referralfrom": "Referuar nga",
     "referralto": "Referohet tek",
     "newpatient": "Lloji i frekuentimit",
-    "newadmittance": "Pranim i ri",
+    "newattendance": "Pranim i ri",
```
Comment (mwithi) [inline src/resources/i18n/sq.json:181]: When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```

Comment (mwithi) [inline src/resources/i18n/sq.json:181]: When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**